themes/startwords/README.md

```diff
@@ -306,31 +306,25 @@ authors:
 
 ## Generating PDFs
 
-PDF versions of feature articles should be created with [paged.js](https://pagedjs.org/) from the production site so that URLs are correct.
+PDF versions of feature articles should be created with [DocRaptor](https://docraptor.com/) from the production site so that URLs are correct.
 
-Install paged.js command-line interface:
-```
-npm install -g pagedjs-cli pagedjs
-```
+First, install python dependencies: `pip install docraptor bs4 requests`. Next, create a DocRaptor account and get an API key. Your API key can be found on your [account dashboard](https://docraptor.com/login). 
```

Comment:
wonder if we need a requirements file; for some reason it didn't bother me putting the requirements in the script - but putting them here is enough removed from the code that it would be harder to remember to update if you add a new dependency